src/AstToCStar.ml

```diff
@@ -242,6 +242,8 @@ let rec mk_expr env in_stmt e =
       CStar.BufRead (mk_expr env e1, mk_expr env e2)
   | EBufSub (e1, e2) ->
       CStar.BufSub (mk_expr env e1, mk_expr env e2)
+  | EBufDiff (e1, e2) ->
+      CStar.BufDiff (mk_expr env e1, mk_expr env e2)
```

Comment:
same here, CStar is untyped, so why not go directly to a subtraction here?

Reply:
with the understanding that there is no need for division since the C compiler will do automatically